src/rpc/mining.cpp

```diff
@@ -53,17 +53,22 @@ using node::UpdateTime;
  * If 'height' is nonnegative, compute the estimate at the time when a given block was found.
  */
 static UniValue GetNetworkHashPS(int lookup, int height, const CChain& active_chain) {
+    if (height > active_chain.Height()) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Block does not exist at specified height");
+    }
+
+    if (lookup < -1 || lookup == 0) {
```

Comment:
You mean the string: "Pass in [blocks] to override # of blocks, -1 specifies since last difficulty change." ?

Seem to me that numbers less than -1 are undefined and 0 is undefined because it results in there being no range for the estimate.

Reply:
Yep, I've changed them to match the RPC help documentation